### ISSUE TITLE
missing hower states on fill ,strone and bg  buttons 

### ISSUE THREAD

**Issue by adambelis** [friction2d/friction#351]
1. open fricion 
2. hover over this buttons
![image](https://github.com/user-attachments/assets/ce27c0b1-f492-475f-a843-dfbb9254ca7a)

there is no hover effect user does not  have feebcakt that this are clickable buttons 
bonus points if the label and icon were included as a clickable button 
hover state
![image](https://github.com/user-attachments/assets/a61ac26f-cace-45e1-98e3-ed2935a81140)

clicked state 
![image](https://github.com/user-attachments/assets/3745c14f-915c-49f6-b53f-ec8e35a132f9)

**Comment by rodlie** [comment]
Sure, I also assume we need something to indicate that you can change the color while hovering with:

- scroll = change hue
- scroll + shift = change value
- scroll + ctrl = change saturation

**Comment by adambelis** [comment]
Oh yes  that's not very discoverable. Maybe as a tooltip or infobar. On hover

**Comment by pgilfernandez** [comment]
May I have a look at this one?

**Comment by rodlie** [comment]
> May I have a look at this one?

 I don't feel like refactoring the toolbutton right now ;)

Feel free, but from my POV there are more important things to fix before release. IMHO this issue should be moved to 1.1.

**Comment by pgilfernandez** [comment]
I've been looking at this for 2 hours and I'm unable to style it... as you say it would need to refactor the whole toolbutton to allow changing it's style... so, I give up

**Comment by rodlie** [comment]
You can't style it, I use custom painting. I know what the problem is, but I didn't design it with hover state in mind. So some work is needed.

**Comment by pgilfernandez** [comment]
BTW, what other important things are pending to be fixed before release? I could have a look and help...

**Comment by rodlie** [comment]
> BTW, what other important things are pending to be fixed before release? 

* #204
* #167
* #300
* Docs

https://github.com/friction2d/friction/milestone/10

**Comment by pgilfernandez** [comment]
yeah, I was looking at [milestone 1.0.0](https://github.com/friction2d/friction/milestone/10).
I might be able to fix #300, if not I will focus on Docs as I sometimes said I was gonna do but ended up doing other stuff...

Thanks

**Comment by rodlie** [comment]
My current plan:

#### Beta 3 (~Monday)

* Update the SDKs
* Merge open PRs tagged 1.0.0

#### RC 1

* Check for regressions
* Try to fix open issues tagged as bug for milestone 1.0.0
* Write basic docs
* Write full release notes for 1.0.0
* Review open issues for 1.0.0, move what we can't fix to milestone 1.1.0
